compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUVectorDistribute.cpp

```diff
@@ -362,7 +362,17 @@ struct LLVMGPUVectorDistributePass
         workgroupSize[i] = llvm::cast<IntegerAttr>(size).getInt();
       }
     } else {
```

Comment:
self note: The logic to get the workgroup size from the function here was a placeholder prior to this PR to avoid needing to specify the whole variant in tests. Drop this in a follow up.

Reply:
Are there tests that do this... I can change them right now.

Reply:
All of these ones: https://github.com/openxla/iree/blob/main/compiler/src/iree/compiler/Codegen/LLVMGPU/test/vector_distribute_layout.mlir + https://github.com/openxla/iree/blob/main/compiler/src/iree/compiler/Codegen/LLVMGPU/test/vector_distribute_conversion.mlir

I don't care either way if it's addressed here or in a follow up though.